src/main/scala/io/shiftleft/js2cpg/preprocessing/TranspilerGroup.scala

```diff
@@ -30,12 +30,12 @@ case class TranspilerGroup(override val config: Config,
 
   private def installPlugins(): Boolean = {
     val command = if (yarnAvailable()) {
-      s"yarn add $BABEL_PLUGINS --dev -W && ${TranspilingEnvironment.YARN_INSTALL}"
+      s"yarn add $BABEL_PLUGINS --dev -W --legacy-peer-deps && ${TranspilingEnvironment.YARN_INSTALL}"
     } else {
-      s"npm install --save-dev $BABEL_PLUGINS && ${TranspilingEnvironment.NPM_INSTALL}"
+      s"npm install --save-dev $BABEL_PLUGINS --legacy-peer-deps && ${TranspilingEnvironment.NPM_INSTALL}"
     }
     logger.info("Installing project dependencies and plugins. This might take a while.")
-    logger.debug("\t+ Installing plugins ...")
+    logger.debug(s"\t+ Installing plugins with command '$command' in path '$projectPath'")
     ExternalCommand.run(command, projectPath.toString) match {
```

Comment:
Can we please also raise the log-level below when the installation fails? To make the error visible to `sl` users.

(Github doesn't let me comment/suggest on that line because it hasn't changed.)